packages/lib-user/webpack.dev.js

```diff
@@ -0,0 +1,85 @@
+const { execSync } = require('child_process')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const path = require('path')
+const webpack = require('webpack')
+
+function gitCommit() {
+  try {
+    const commitHash = execSync('git describe --always').toString('utf8').trim()
+    return commitHash
+  } catch (error) {
+    console.log(error)
+    return 'Not a git repository.'
+  }
+}
+
+const EnvironmentWebpackPlugin = new webpack.EnvironmentPlugin({
+  COMMIT_ID: gitCommit(),
+  DEBUG: false,
+  NODE_ENV: 'development',
+  PANOPTES_ENV: 'staging'
+})
+
+const HtmlWebpackPluginConfig = new HtmlWebpackPlugin({
+  template: './dev/index.html',
+  filename: 'index.html',
+  inject: 'body'
+})
+
+module.exports = {
+  devServer: {
```

Comment:
This is running on port 8080, the default and same as lib-classifier, but should it be changed so both lib-classifier and lib-user can be ran locally at the same time?

Reply:
We run both NextJS apps on port 3000 so 🤷🏻‍♂️.